src/app/global.css

```
@@ -714,11 +744,32 @@ https://velog.io/@oneook/tailwindcss-4.0-%EB%AC%B4%EC%97%87%EC%9D%B4-%EB%8B%AC%E
+  line-height: 39px;
+  letter-spacing: -0.494px;
+}
+
+@utility font-designer-26b {
+  font-size: 26px;
+  font-weight: var(--font-weight-bold);
+  line-height: 39px;
+  letter-spacing: -0.494px;
+}
+
+@utility font-designer-26r {
```

Stylelint (17.11.0)

[error] Unexpected unknown at-rule "@utility" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)